<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergedScreen</class>
 <widget class="QWidget" name="MergedScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">QLabel{color: rgb(255, 255, 255);}</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>885</width>
        <height>511</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QGridLayout" name="securities">
         <item row="0" column="0">
          <layout class="QGridLayout" name="transactions">
           <property name="horizontalSpacing">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>10</number>
           </property>
           <item row="0" column="2">
            <widget class="QLabel" name="buy">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(128, 162, 255);
color: rgb(255, 255, 255);
border: 3px solid rgb(74, 74, 74);</string>
             </property>
             <property name="text">
              <string>Buy</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="sell">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(128, 162, 255);
color: rgb(255, 255, 255);
border: 3px solid rgb(74, 74, 74);</string>
             </property>
             <property name="text">
              <string>Sell</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="price">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(128, 162, 255);
color: rgb(255, 255, 255);
border: 3px solid rgb(74, 74, 74);</string>
             </property>
             <property name="text">
              <string>Price</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="cost">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(128, 162, 255);
color: rgb(255, 255, 255);
border: 3px solid rgb(74, 74, 74);</string>
             </property>
             <property name="text">
              <string>Cost</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="9">
            <widget class="QLabel" name="bookValue">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(128, 162, 255);
color: rgb(255, 255, 255);
border: 3px solid rgb(74, 74, 74);</string>
             </property>
             <property name="text">
              <string>Book Value</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="proceeds">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(128, 162, 255);
color: rgb(255, 255, 255);
border: 3px solid rgb(74, 74, 74);</string>
             </property>
             <property name="text">
              <string>Proceeds</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="8">
            <widget class="QLabel" name="commissions">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(128, 162, 255);
color: rgb(255, 255, 255);
border: 3px solid rgb(74, 74, 74);</string>
             </property>
             <property name="text">
              <string>ACQ/DIS BV</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="11">
            <widget class="QLabel" name="gainLoss">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(128, 162, 255);
color: rgb(255, 255, 255);
border: 3px solid rgb(74, 74, 74);</string>
             </property>
             <property name="text">
              <string>Gain/Loss</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="description">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(128, 162, 255);
color: rgb(255, 255, 255);
border: 3px solid rgb(74, 74, 74);</string>
             </property>
             <property name="text">
              <string>Description</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="date">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(128, 162, 255);
color: rgb(255, 255, 255);
border: 3px solid rgb(74, 74, 74);</string>
             </property>
             <property name="text">
              <string>Date</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="10">
            <widget class="QLabel" name="avgCB">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(128, 162, 255);
color: rgb(255, 255, 255);
border: 3px solid rgb(74, 74, 74);</string>
             </property>
             <property name="text">
              <string>Average Base Cost</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QLabel" name="totalShares">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(128, 162, 255);
color: rgb(255, 255, 255);
border: 3px solid rgb(74, 74, 74);</string>
             </property>
             <property name="text">
              <string>Total Shares</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="back">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(208, 208, 208);</string>
     </property>
     <property name="text">
      <string>Back To Main Menu</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
